scipy/spatial/_spherical_voronoi.py

```diff
@@ -141,33 +129,31 @@ class SphericalVoronoi:
 
     """
 
-    def __init__(self, points, radius=1, center=(0, 0, 0), threshold=1e-06):
+    def __init__(self, points, radius=1, center=None, threshold=1e-06):
```

Comment:
I had to change the default back to `center=None` :)
I can't see a way to set the default to all zeros when the input is variable length.  Any suggestions?

Reply:
Not off the top of my head, and I think Pauli recommended avoiding mutable kwargs in your previous PR to adjust the API.

One thing for us to keep in mind is that if the API change / deprecation from your previous PR goes out in 1.4 and this restoration doesn't happen until later, that might be a little bit awkward from the user experience standpoint I suppose. I'll put a 1.4 milestone on this PR just so we are reminded to come back to this before the release if this PR is still open.